### DISCUSSION TITLE
Why there is no documentation regarding the host-gateway IP?

### DISCUSSION THREAD

**Discussion by jkilzi** [containers/podman#22183] [General]
This is the situation right now:
- Docker docs: https://docs.docker.com/reference/cli/dockerd/#configure-host-gateway-ip
- Podman docs: https://docs.podman.io/en/latest/search.html?q=host-gateway (not a single word about it)
As a newcomer, in order to learn about this concept I had to read the first article. [If this is supported now by Podman](https://github.com/containers/podman/issues/14390) why there is not a single word about it in the docs? Did I miss something?

**Comment by rhatdan** [comment]
Interested in opening a PR to add it?
@Luap99 PTAL

**Comment by Luap99** [comment]
Not sure what to say, a lot of things are not documented. If the contributor for a feature does not add docs then there are no docs for it. That is the reality most likely.
